Formula/r/ratchet.rb

```diff
@@ -19,14 +20,24 @@ class Ratchet < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args(ldflags: "-s -w")
+    ldflags = %W[
+      -s
+      -w
+      -X=github.com/sethvargo/ratchet/internal/version.version=#{version}
+      -X=github.com/sethvargo/ratchet/internal/version.commit=#{Utils.git_head}
+      -extldflags=-static
+    ]
+    system "go", "build", *std_go_args(ldflags:)
 
     pkgshare.install "testdata"
   end
 
   test do
-    cp_r pkgshare/"testdata", testpath
-    output = shell_output(bin/"ratchet check testdata/github.yml 2>&1", 1)
+    cp_r "#{pkgshare}/testdata", testpath
```

Comment:
```suggestion
    cp_r pkgshare/"testdata", testpath
```
You don't need to change this one.

Reply:
Done